pycbc/filter/qtransform.py

```diff
@@ -216,16 +216,16 @@ def qseries(fseries, Q, f0, return_raw=False):
 
     tlen = (len(fseries)-1) * 2
     windowed.resize(tlen)
-    wenergy = numpy.roll(windowed, -center)
+    windowed = numpy.roll(windowed, -center)
 
     # calculate the time series for this q -value
-    wenergy = FrequencySeries(wenergy, delta_f=fseries.delta_f,
+    windowed = FrequencySeries(windowed, delta_f=fseries.delta_f,
                             epoch=fseries.start_time)
     cenergy = TimeSeries(zeros(tlen, dtype=numpy.complex128),
```

Comment:
and as I said in the last line of the comment above `cenergy` -> `ctseries` or something similar as it's not energy.  or eg `cwave` as it's the complex wavelet transform of the original timeseries

Reply:
Ok, done.